packages/mui-material/src/Autocomplete/Autocomplete.spec.tsx

```diff
@@ -66,6 +66,15 @@ function MyAutocomplete<
   renderInput={() => null}
 />;
 
+// Tests presence of onMouseDown prop in InputProps
+<Autocomplete
+  options={['1', '2', '3']}
+  renderInput={(params) => {
+    console.log(params.InputProps.onMouseDown);
```

Comment:
I guess we could also use `expectType` instead of `console.log`. If we don't care about the actual type and it's covered with unit tests I guess `console.log` it's fine.

Reply:
I guess it's better to use expectType, we can be bit strict with tests and flexible in implementaion. updated here https://github.com/mui/material-ui/pull/44183/commits/8e4626475e89552be3dda9f4125694e67e63dc5c